### PR TITLE
Updated ReadMe.md

### PR DESCRIPTION
**Pull request by MadeFaux**
Changes to the README.md that Sophia's batch as made!

### CODE REVIEW

**Review by thomaspceci — DISMISSED** [review]
Looks good

File: README.md [hunk 1]
```diff
@@ -30,22 +30,16 @@ The models used to create the datatables are as follows.
     - This model contains the iformation regarding the project done while Working at Revature.
```
Comment (sophiagavrila) [inline README.md]: typo on "information"

**Review by ajang06 — DISMISSED** [review]
lgtm

File: README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-
-## What does the backend consist of?
-The backend consists of the models for each subsection of a given portfolio driven by the Spring Framework as well as the necessary functions to support creating, updating, and deleting sections of the portfolio. Each subsection of a given portfolio can be updated independently without affecting the other sections of the portfolio which prevents minor changes to a specific function from causing a breakdown across the entire application. The database embedded within the back end is a H2 database which allows for the system to run without a PostgreSQL database already running at startup.
+The Forge Portfolio Rework is an overhaul of Revature's portfolio system on their website. This rework is based on refactoring code that a previous batch had created.
```
Comment (sophiagavrila) [inline README.md]: Include some general information about what Forge does aside from it being an overhaul of Revature's current portfolio system. You can retain some of the previous verbiage: 

perhaps include: _"This system aims to create a new portfolio system that is both more intuitive for the user and makes it easier to understand the requirements that Revature has in place for them."_

**Review by sophiagavrila — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (sophiagavrila) [reply README.md]: Eventually include a `Contributors` section with all names

**Review by sophiagavrila — APPROVED** [review]

**Review by FBThornton — APPROVED** [review]

**Pull request merged**